test/hummingbot/connector/gateway/amm/debug_gateway_cancel.py

```diff
@@ -91,7 +91,7 @@ async def collect_testing_data(self):
         await self.collect_cancel_approval()
 
     async def collect_cancel_order(self):
-        print("Creating and then canceling Uniswap order...\t\t", end="", flush=True)
+        print("Creating and then canceling Amm order...\t\t", end="", flush=True)
```

Comment:
We have connector / exchange named uniswap
https://hummingbot.org/exchanges/uniswap/

this changes here not needed